cmd/controller-manager/app/controller-manager.go

```diff
@@ -113,15 +114,15 @@ func Run(opts *options.Options) error {
 	setOptionsByFederationConfig(opts)
 
 	if err := utilfeature.DefaultFeatureGate.SetFromMap(opts.FeatureGates); err != nil {
-		glog.Fatalf("Invalid Feature Gate: %v", err)
+		klog.Fatalf("Invalid Feature Gate: %v", err)
```

Comment:
@font, i think you missed this comment from the @poothia's PR. https://github.com/kubernetes-sigs/federation-v2/pull/695#issuecomment-477451066 . what is your opinion? I think it would reduce much of the unwanted code change.

Reply:
I believe you're referring to using an alias. Having an alias would certainly reduce the number of code changes to just the header imports of each file that uses `glog`. But this is mostly a mechanical change (aside from dependency changes) so I'm not clear on the desire to not want the complete code change if it's already done. Plus using an alias makes `goimports` not work as well when you're trying to add a new import to a file you're editing.

Reply:
I was concerned about the downstream users having to absorb all the changes which could be avoidable. But anyway its minor effort as you said. I will leave it to @marun to LGTM please. Thanks !